src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -468,14 +490,15 @@ def _alloc_nompi(self, unit):
             # attempt to find the required number of cores and gpus on this
             # node - do not allow partial matches.
             cores, gpus, lfs, mem = self._find_resources(node=node,
-                                                    requested_cores=total_cores,
-                                                    requested_gpus=total_gpus,
-                                                    requested_lfs=total_lfs,
-                                                    requested_mem=total_mem,
-                                                    partial=False,
-                                                    lfs_chunk=lfs_chunk,
-                                                    core_chunk=core_chunk,
-                                                    mem_chunk=mem_chunk)
+                                                         requested_cores=total_cores,
+                                                         requested_gpus=total_gpus,
+                                                         requested_lfs=total_lfs,
+                                                         requested_mem=total_mem,
+                                                         core_chunk=core_chunk,
+                                                         partial=False,
+                                                         gpu_chunk=1,
```

Comment:
Why is `gpu_chunk` set to 1?

Reply:
because we do not yet really have anything like GPU threads (and per our discussion on task description its unclear if that will ever happen), so we have a different co-locality notion than for cores (where threads require multiple cores in one chunk, i.e. on the same node).